07.SubpassBaking/main.cpp

```diff
@@ -410,8 +413,8 @@ class SubpassBaking : public ApplicationBase
                 auto upQueue = queues[decltype(initOutput)::EQT_TRANSFER_UP];
                 auto fence = logicalDevice->createFence(video::IGPUFence::ECF_UNSIGNALED);
                 core::smart_refctd_ptr<video::IGPUCommandBuffer> tferCmdBuf;
-                logicalDevice->createCommandBuffers(commandPools[decltype(initOutput)::EQT_TRANSFER_UP].get(), video::IGPUCommandBuffer::EL_PRIMARY, 1u, &tferCmdBuf);
-                tferCmdBuf->begin(IGPUCommandBuffer::EU_NONE); // TODO some one time submit bit or something
+                logicalDevice->createCommandBuffers(commandPools[decltype(initOutput)::EQT_TRANSFER_UP][0].get(), video::IGPUCommandBuffer::EL_PRIMARY, 1u, &tferCmdBuf);
```

Comment:
better name these [0] indices -> defaultXXXCommandPool which is basically for pre-render loop stuff and for frame 0

Please apply it to other examples, I won't comment this on every place